create.go

```diff
@@ -284,3 +292,58 @@ func createPIDFile(pidFilePath string, pid int) error {
 
 	return nil
 }
+
+// copyIfNeeded copies the cpuset.cpus and cpuset.mems from the parent
+// directory to the current directory if the file's contents are 0
+func copyIfNeeded(current, parent string) error {
+	var (
+		err                      error
+		currentCpus, currentMems []byte
+		parentCpus, parentMems   []byte
+	)
+
+	if currentCpus, currentMems, err = getSubsystemSettings(current); err != nil {
+		return err
+	}
+	if parentCpus, parentMems, err = getSubsystemSettings(parent); err != nil {
+		return err
+	}
+
+	if isEmpty(currentCpus) {
+		if err := writeFile(current, "cpuset.cpus", string(parentCpus)); err != nil {
+			return err
+		}
+	}
+	if isEmpty(currentMems) {
+		if err := writeFile(current, "cpuset.mems", string(parentMems)); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func getSubsystemSettings(parent string) (cpus []byte, mems []byte, err error) {
```

Comment:
Missing function header to explain what this does. Also, I'd prefer the name to container `cgroup` to make it clearer what it is operating on.